cw-dashboard-lambda-metrics/src/lambda/proxy-lambda/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM public.ecr.aws/lambda/dotnet:6 AS base
+
+FROM mcr.microsoft.com/dotnet/sdk:6.0-bullseye-slim as build
+WORKDIR /src
+COPY ["proxy-lambda.csproj", "proxy-lambda/"]
+RUN dotnet restore "proxy-lambda/proxy-lambda.csproj"
+
+WORKDIR "/proxy-lambda"
+COPY . .
+RUN dotnet build "proxy-lambda.csproj" --configuration Release --output /app/build
+
+FROM build AS publish
+RUN dotnet publish "proxy-lambda.csproj" \
+            --configuration Release \ 
+            --runtime linux-x64 \
+            --self-contained false \ 
+            --output /app/publish \
+            -p:PublishReadyToRun=true  
+
+FROM base AS final
+RUN curl -O https://lambda-insights-extension.s3-ap-northeast-1.amazonaws.com/amazon_linux/lambda-insights-extension.rpm && \
```

Comment:
you are fixing the extension region here and if you deploy from a different region fails.you need to make this dynamic.

Reply:
I have made it as a ARG in dockerfile and passing it from the Stack.